waiter/src/waiter/token.clj

```diff
@@ -69,18 +83,22 @@
 
   (defn delete-service-description-for-token
     "Delete a token from the KV"
-    [clock synchronize-fn kv-store token owner & {:keys [hard-delete] :or {hard-delete false}}]
+    [clock synchronize-fn kv-store token owner &
+     {:keys [hard-delete update-time] :or {hard-delete false, update-time Long/MAX_VALUE}}]
```

Comment:
`if-unmodified-since` would be a better name.  `update-time` implies that is the timestamp that should be written for `last-update-time`.

Reply:
Done.